php_phongo.h

```diff
@@ -110,7 +110,6 @@ void              phongo_throw_exception(php_phongo_error_domain_t domain TSRMLS
 #endif /* PHP_WIN32 */
 #endif /* PHP_VERSION_ID < 70000 */
 	;
-void phongo_throw_exception_from_bson_error_t(bson_error_t* error TSRMLS_DC);
```

Comment:
> whether it's wise to get rid of that older API that doesn't take a reply_t as well.

I think it'd be best to keep `phongo_throw_exception_from_bson_error_t()` and simply have it call `phongo_throw_exception_from_bson_error_and_reply_t()` with a `NULL` reply. I imagine it'd be a quick compiler optimization and it's also cut down on many of the code replacements in this PR.